keycloak/keycloak-ui
# Supporting EdDSA

closes #4437

## Motivation
Shown in https://github.com/keycloak/keycloak/issues/15714 .

## Brief Description
Shown in https://github.com/keycloak/keycloak-ui/issues/4437

## Verification Steps
1. move to Realm settings > Tokens and assure no error occurs.
2. move to Realm settings > Keys > Providers and create eddsa-generated provider by "Add provider"
3. move to Realm settings > Keys > Keys list and assure that the created key is shown.

## Additional Notes
Both the PR in keycloak repository (https://github.com/keycloak/keycloak/pull/17215) and the PR in keycloak-ui must be merged simultaneously.
